packages/react-utils/src/compose/renderShorthandChildren.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { ObjectShorthandProps, ShorthandRenderFunction } from './types';
+
+/**
+ * A helper function to resolve the correct kind of shorthand children to render
+ */
+export const renderShorthandChildren = <TProps,>(
```

Comment:
**chore:** can we add test for this (you're adding new public API)

Reply:
We had an offline discussion with @ling1726, we don't need this, see `25ef50c`. @ling1726 can you please remove this util?

Reply:
removed